src/System.Windows.Forms/tests/UnitTests/CheckedListBoxTests.cs

```diff
@@ -594,10 +625,242 @@ public void CheckedListBox_Remove_CheckedItems(bool createControl)
+    {
+        using SubCheckedListBox checkedListBox = new();
+        int expectedHeight = checkedListBox.Font.Height + checkedListBox._listItemBordersHeight;
+
+        checkedListBox.ItemHeight.Should().Be(expectedHeight);
+    }
+
+    [WinFormsFact]
+    public void CheckedListBox_ItemHeight_Set_DoesNotChangeValue()
+    {
+        using CheckedListBox checkedListBox = new();
+        int originalHeight = checkedListBox.ItemHeight;
+
+        checkedListBox.ItemHeight = 50;
+        checkedListBox.ItemHeight.Should().Be(originalHeight);
+
+        checkedListBox.ItemHeight = -1;
+        checkedListBox.ItemHeight.Should().Be(originalHeight);
+    }
+
+    [WinFormsTheory]
+    [BoolData]
+    public void CheckedListBox_UseCompatibleTextRendering_GetSet_ReturnsExpected(bool value)
+    {
+        using CheckedListBox checkedListBox = new();
+
+        checkedListBox.UseCompatibleTextRendering = value;
+        checkedListBox.UseCompatibleTextRendering.Should().Be(value);
+        checkedListBox.UseCompatibleTextRenderingInternal.Should().Be(value);
+    }
+
+    [WinFormsFact]
+    public void CheckedListBox_DataSourceChanged_AddRemove_Success()
+    {
+        using SubCheckedListBox checkedListBox = new();
+        int callCount = 0;
+        EventHandler handler = (sender, e) =>
+        {
+            callCount++;
+            sender.Should().Be(checkedListBox);
+            e.Should().Be(EventArgs.Empty);
+        };
+
+        checkedListBox.DataSourceChanged += handler;
+        checkedListBox.OnDataSourceChanged(EventArgs.Empty);
+        callCount.Should().Be(1);
+
+        checkedListBox.DataSourceChanged -= handler;
+        checkedListBox.OnDataSourceChanged(EventArgs.Empty);
+        callCount.Should().Be(1);
+    }
+
+    [WinFormsFact]
+    public void CheckedListBox_DisplayMemberChanged_AddRemove_Success()
+    {
+        using SubCheckedListBox checkedListBox = new();
+        int callCount = 0;
+        EventHandler handler = (sender, e) =>
+        {
+            callCount++;
+            sender.Should().Be(checkedListBox);
+            e.Should().Be(EventArgs.Empty);
+        };
+
+        checkedListBox.DisplayMemberChanged += handler;
+        checkedListBox.OnDisplayMemberChanged(EventArgs.Empty);
+        callCount.Should().Be(1);
+
+        checkedListBox.DisplayMemberChanged -= handler;
+        checkedListBox.OnDisplayMemberChanged(EventArgs.Empty);
+        callCount.Should().Be(1);
+    }
+
+    [WinFormsFact]
+    public void CheckedListBox_ItemCheck_AddRemove_Success()
+    {
+        using SubCheckedListBox checkedListBox = new();
+        int callCount = 0;
+        ItemCheckEventHandler handler = (sender, e) =>
+        {
+            callCount++;
+            sender.Should().Be(checkedListBox);
+            e.Should().NotBeNull();
+        };
+
+        ItemCheckEventArgs eventArgs = new ItemCheckEventArgs(0, CheckState.Checked, CheckState.Unchecked);
```

Comment:
Please use target type new
```suggestion
        ItemCheckEventArgs eventArgs = new(0, CheckState.Checked, CheckState.Unchecked);
```